ngraph/test/runtime/ie/unit_test.manifest

```diff
@@ -40,6 +40,8 @@ onnx_model_matmul_integer_4d_zero_point
 onnx_model_matmul_integer_4d_no_zero_point
 onnx_model_qlinear_matmul
 onnx_model_qlinear_matmul_3d
+onnx_model_gather_elements_int32
+onnx_model_gather_elements_float
```

Comment:
Are these tests really failing with `# Unsupported operator detected in the graph: QuantizedDot` as the comment above (line 33) says?

Reply:
No, these aren't. I'll correct that.